ios/FluentUI/Resources/Localization/ar.lproj/Localizable.strings

```diff
@@ -1,205 +0,0 @@
-//
```

Comment:
Not sure what triggered this PR. I'll investigate shortly, but may end up just abandoning this PR since I don't imagine we actually wanted to delete all our localized files.